trello_items.py

```diff
@@ -0,0 +1,110 @@
+import requests
+import secrets
+from card import Card
+
+key = secrets.apiKey
+token = secrets.apiToken
+
+def get_todo_cards_for_board():
+    """
+    Fetches all To Do cards from the board.
+
+    Returns:
+        list: The list of cards.
+    """
+
+    query = {
+        'key': key,
+        'token': token
+    }
+   
+    board_id = secrets.boardId
+    to_do_cards = []
+    url = "https://api.trello.com/1/boards/{}/cards".format(board_id)
+    cards = requests.request("GET", url, params=query)
+    for card in cards.json():
+        if (get_list_name(card['id']) == 'To Do'):
+            to_do_cards.append(Card(card['id'], card['pos'], card['name'], card['desc'], get_list_name(card['id'])))
+    return to_do_cards
+
+def get_cards_for_board():
+    """
+    Fetches all To Do cards from the board.
```

Comment:
This description is exactly the same as the previous function "get_todo_cards_for_board".
Did you forget to update it?

Also, there's duplicated code. 

But most importantly, do you want to just have this function and delete the other function? Currently only "To Do" items are displayed on your app - it's not possible to view done cards or move them back to "to do"

Reply:
Done